src/main.js

```diff
@@ -52,33 +60,52 @@ function displayCurrentCover() {
 
 function displayFormView() {
   homeViewSection.classList.add("hidden");
-  savedViewSection.classList.add("hidden"); 
-  formViewSection.classList.remove("hidden");  
-  randomCoverButton.classList.add("hidden"); 
+  savedViewSection.classList.add("hidden");
+  formViewSection.classList.remove("hidden");
+  randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
 //*************ADDED code BELOW to ITER2 ********
   viewSavedButton.classList.remove("hidden");
+  makeYourOwnButton.classList.add("hidden");
+  //hide makeyourownbutton
 }
-   
+
 function displaySavedView() {
-  savedViewSection.classList.remove("hidden"); 
+  savedViewSection.classList.remove("hidden");
   homeViewSection.classList.add("hidden");
-  formViewSection.classList.add("hidden");  
-  randomCoverButton.classList.add("hidden"); 
+  formViewSection.classList.add("hidden");
+  randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
 //*************ADDED code BELOW to ITER2 ********
-  viewSavedButton.classList.add("hidden"); 
+  viewSavedButton.classList.add("hidden");
+  makeYourOwnButton.classList.remove("hidden");
```

Comment:
I concur, if we hide it while in form view, guess we have to show it when in Saved View!